.env

```diff
@@ -40,3 +40,4 @@ REACT_APP_AUTH_OIDC_PUBLIC_CLIENT=
 REACT_APP_AUTH_OIDC_CONFIDENTIAL_PORT=0
 # App config
 REACT_APP_CONTACT=
+REACT_APP_HEADBAND=
```

Comment:
Not sure if `HEADBAND` is meaningful enough. I suggest using something like `ALERT_MESSAGE` ? Or something like that ? What do you think ?